tests/themes/run.sh

```diff
@@ -23,6 +23,8 @@ for theme in themes/*/theme.lua; do
 
   sed "s~default/theme~$theme/theme~g" "awesomerc.lua" > "$config_file"
 
+  # Set CMAKE_BINARY_DIR for out-of-tree builds.
+  CMAKE_BINARY_DIR="$PWD" \
```

Comment:
Is this necessary here? Shouldn't this be done in `tests/run.sh` instead? What uses this env var anyway? I don't think anything here is using CMake?